### ISSUE TITLE
Android TV support 

### ISSUE THREAD

**Issue by VasilisKos** [PranshulGG/WeatherMaster#119]
Would be great.

**Comment by PranshulGG** [comment]
If I get it to work, then yes!

**Comment by ACR-Jeff** [comment]
Yes this would be awesome and a game changer for your app IMO, I've tried it on my nvidia shield and the issue I was facing was navigations within the UI, No way to tell where navigating, No place to click to change options with the D-pad, or controllers, Even tried with a touch pad, Didn't function too well.

Maybe changing color of item when selected to change/apply
